modules/storage-persistent-storage-pv.adoc

```diff
@@ -0,0 +1,293 @@
+|===
+
+[IMPORTANT]
+====
+A volume's `AccessModes` are descriptors of the volume's capabilities. They
+are not enforced constraints. The storage provider is responsible for 
+runtime errors resulting from invalid use of the resource.
+
+For example, Ceph offers *ReadWriteOnce* access mode. You must
+mark the claims as `read-only` if you want to use the volume's
+ROX capability. Errors in the provider show up at runtime as mount errors.
+
+iSCSI and Fibre Channel volumes do not currently have any fencing 
+mechanisms. You must ensure the volumes are only used by one node at a 
+time. In certain situations, such as draining a node, the volumes can be 
+used simultaneously by two nodes. Before draining the node, first ensure 
+the pods that use these volumes are deleted.
+====
+
+.Supported access modes for PVs
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume Plug-in  |ReadWriteOnce  |ReadOnlyMany  |ReadWriteMany
+|AWS EBS  | ✅ | - |  -
+|Azure File | ✅ | ✅ | ✅
+|Azure Disk | ✅ | - | -
+|Ceph RBD  | ✅ | ✅ |  -
+|Fibre Channel  | ✅ | ✅ |  -
+|GCE Persistent Disk  | ✅ | - |  -
+|GlusterFS  | ✅ | ✅ | ✅
+|HostPath  | ✅ | - |  -
+|iSCSI  | ✅ | ✅ |  -
+|NFS  | ✅ | ✅ | ✅
+|Openstack Cinder  | ✅ | - |  -
+|VMWare vSphere | ✅ | - |  -
+|Local | ✅ | - |  -
+|===
+
+[NOTE]
+====
+Use a recreate deployment strategy for pods that rely on AWS EBS.
+// GCE Persistent Disks, or Openstack Cinder PVs. 
+====
+
+
+ifdef::openshift-dedicated,openshift-online[]
+[[pv-restrictions]]
+== Restrictions
+
+The following restrictions apply when using PVs with {product-title}:
+endif::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+ * PVs are provisioned with either EBS volumes (AWS) or GCP storage (GCP), 
+depen ding on where the cluster is provisioned.
+ * Only RWO access mode is applicable, as EBS volumes and GCE Persistent 
+Disks cannot be mounted to multiple nodes.
+ * *emptyDir* has the same lifecycle as the pod:
+   ** *emptyDir* volumes survive container crashes/restarts.
+   ** *emptyDir* volumes are deleted when the pod is deleted.
+====
+endif::[]
+
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+ * PVs are provisioned with EBS volumes (AWS).
+ * Only RWO access mode is applicable, as EBS volumes and GCE Persistent 
+Disks cannot be mounted to multiple nodes.
+ * Docker volumes are disabled.
+   ** VOLUME directive without a mapped external volume fails to be 
+instantiated
+.  
+ * *emptyDir* is restricted to 512 Mi per project (group) per node.
+   ** A single pod for a project on a particular node can use up to 512 Mi 
+of *emptyDir* storage.
+   ** Multiple pods for a project on a particular node share the 512 Mi of 
+*emptyDir* storage. 
+ *  *emptyDir* has the same lifecycle as the pod:
+   ** *emptyDir* volumes survive container crashes/restarts.
+   ** *emptyDir* volumes are deleted when the pod is deleted.
+====
+endif::[]
+
+[[pv-reclaim-policy]]
+== Reclaim policy
+
+The following table lists current reclaim policies:
```

Comment:
There is only one policy, should this just say policy? (same with the table title below)

Reply:
I've adjusted this to be singular for now. If additional policies are included I'll change it back.